Error in nokogumbo when using Ruby 2.7.0

I think you have to update nokogiri to 1.11.0 according to https://github.com/msys2/MINGW-packages/issues/6222 .
Thanks.

```
Using jekyll-toc 0.17.0
Installing nokogumbo 2.0.5 with native extensions
Gem::Ext::BuildError: ERROR: Failed to build gem native extension.

    current directory: /vendor/bundle/ruby/2.7.0/gems/nokogumbo-2.0.5/ext/nokogumbo
/usr/bin/ruby2.7 -I /usr/lib/ruby/2.7.0 -r ./siteconf20210509-13013-193agq5.rb extconf.rb
Ignoring nokogumbo-2.0.5 because its extensions are not built. Try: gem pristine nokogumbo --version 2.0.5
*** extconf.rb failed ***
Could not create Makefile due to some reason, probably lack of necessary
libraries and/or headers.  Check the mkmf.log file for more details.  You may
need configuration options.

Provided configuration options:
        --with-opt-dir
        --without-opt-dir
        --with-opt-include
        --without-opt-include=${opt-dir}/include
        --with-opt-lib
        --without-opt-lib=${opt-dir}/lib
        --with-make-prog
        --without-make-prog
        --srcdir=.
        --curdir
        --ruby=/usr/bin/$(RUBY_BASE_NAME)2.7
/usr/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:155:in `require': cannot load such file -- nokogiri/nokogiri (LoadError)
        from /usr/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:155:in `require'
        from /vendor/bundle/ruby/2.7.0/gems/nokogiri-1.11.3-x64-mingw32/lib/nokogiri/extension.rb:25:in `rescue in <top (required)>'
        from /vendor/bundle/ruby/2.7.0/gems/nokogiri-1.11.3-x64-mingw32/lib/nokogiri/extension.rb:4:in `<top (required)>'
        from /usr/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:103:in `require'
        from /usr/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:103:in `require'
        from /vendor/bundle/ruby/2.7.0/gems/nokogiri-1.11.3-x64-mingw32/lib/nokogiri.rb:11:in `<top (required)>'
        from /usr/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:168:in `require'
        from /usr/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:168:in `rescue in require'
        from /usr/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:156:in `require'
        from extconf.rb:4:in `<main>'
/usr/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:103:in `require': /vendor/bundle/ruby/2.7.0/gems/nokogiri-1.11.3-x64-mingw32/lib/nokogiri/2.7/nokogiri.so: invalid ELF header -
/vendor/bundle/ruby/2.7.0/gems/nokogiri-1.11.3-x64-mingw32/lib/nokogiri/2.7/nokogiri.so (LoadError)
        from /usr/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:103:in `require'
        from /vendor/bundle/ruby/2.7.0/gems/nokogiri-1.11.3-x64-mingw32/lib/nokogiri/extension.rb:6:in `<top (required)>'
        from /usr/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:103:in `require'
        from /usr/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:103:in `require'
        from /vendor/bundle/ruby/2.7.0/gems/nokogiri-1.11.3-x64-mingw32/lib/nokogiri.rb:11:in `<top (required)>'
        from /usr/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:168:in `require'
        from /usr/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:168:in `rescue in require'
        from /usr/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:156:in `require'
        from extconf.rb:4:in `<main>'
/usr/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:92:in `require': cannot load such file -- nokogiri (LoadError)
        from /usr/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:92:in `require'
        from extconf.rb:4:in `<main>'

extconf failed, exit code 1

Gem files will remain installed in /vendor/bundle/ruby/2.7.0/gems/nokogumbo-2.0.5 for inspection.
Results logged to /vendor/bundle/ruby/2.7.0/extensions/x86_64-linux/2.7.0/nokogumbo-2.0.5/gem_make.out

An error occurred while installing nokogumbo (2.0.5), and Bundler cannot continue.
Make sure that `gem install nokogumbo -v '2.0.5' --source 'https://rubygems.org/'` succeeds before bundling.

In Gemfile:
  html-proofer was resolved to 3.19.1, which depends on
    nokogumbo
make: *** [Makefile:28: install] Error 5
```

--- a/.rubocop.yml
+++ b/.rubocop.yml
@@ -1,5 +1,5 @@
 AllCops:
-  TargetRubyVersion: 2.4
+  TargetRubyVersion: 2.5
   NewCops: enable
   Exclude:
     - "*.gemspec"

--- a/jekyll-toc.gemspec
+++ b/jekyll-toc.gemspec
@@ -17,7 +17,7 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec)/})
   spec.require_paths = ['lib']
 
-  spec.required_ruby_version = '>= 2.4'
+  spec.required_ruby_version = '>= 2.5'
 
   spec.add_dependency 'jekyll', '>= 3.9'
   spec.add_dependency 'nokogiri', '~> 1.10'

--- a/jekyll-toc.gemspec
+++ b/jekyll-toc.gemspec
@@ -20,5 +20,5 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = '>= 2.5'
 
   spec.add_dependency 'jekyll', '>= 3.9'
-  spec.add_dependency 'nokogiri', '~> 1.10'
+  spec.add_dependency 'nokogiri', '~> 1.11'
 end

